src/connectors/relisten.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+Connector.currentTimeSelector = '.player div.timing:not(.duration) div:last-child';
+
+Connector.durationSelector = '.player div.timing.duration div:last-child';
+
+Connector.playerSelector = '.navigation > .player';
+
+Connector.trackSelector = '.player .content .info .song-title';
+
+Connector.getAlbum = () => {
```

Comment:
Please use `Connector.getTrackInfo` and `Util.splitString` instead of `Connector.getAlbum` and `Connector.getArtist` in this connector.

You can use the connector for Apple Music as an example.
https://github.com/web-scrobbler/web-scrobbler/blob/c17279c216e8e09e32c56a2604dde11cac33c436/src/connectors/apple-music.js#L17-L24